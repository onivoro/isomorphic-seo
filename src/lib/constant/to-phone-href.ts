export const toPhoneHref = (phoneNumber: string) => `tel:${phoneNumber}`;
