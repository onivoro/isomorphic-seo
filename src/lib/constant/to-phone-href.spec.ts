import { toPhoneHref } from './to-phone-href';
const testPhone = '5555555555';

describe('toPhoneHref', () => {
    describe('success case', () => {
        it('worx', () => {
            expect(toPhoneHref(testPhone))
                .toMatchSnapshot();
        });

    });
});
