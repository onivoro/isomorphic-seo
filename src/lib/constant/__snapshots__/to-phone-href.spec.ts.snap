// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`toPhoneHref success case worx 1`] = `"tel:5555555555"`;
